modules/menu/menu.c

```diff
@@ -456,6 +499,7 @@ static const struct cmd cmdv[] = {
 {NULL,        KEYCODE_ESC,0, "Hangup call",             cmd_hangup           },
 {"uanext",    'T',        0, "Toggle UAs",              cmd_ua_next          },
 {"uanew",     0,    CMD_PRM, "Create User-Agent",       create_ua            },
+{"uafind",    0,    CMD_PRM, "Find UA",                 cmd_ua_find          },
```

Comment:
suggestion for description:

```
"Find User-Agent <aor>"
```

(so the user can see that the expected input is a SIP aor).